test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -2775,13 +2774,13 @@ private Text getRandomText(final int len) {
   }
 
   private static String encode(final Text text, final boolean encode) {
-    if (StringUtils.isBlank(text.toString()))
+    if (text.toString().isBlank())
       return null;
     return encode ? Base64.getEncoder().encodeToString(TextUtil.getBytes(text)) : text.toString();
   }
 
   private Text decode(final String text, final boolean decode) {
-    if (StringUtils.isBlank(text))
+    if (text.isBlank())
```

Comment:
Not sure if all callers pass non-null, but if the callers aren't careful, this could be null. Could verify with:

```suggestion
    if (requireNonNull(text).isBlank())
```

(with a static import of `Objects.requireNonNull`)